src/lib/PreviewUI.js

```diff
@@ -261,10 +261,12 @@ class PreviewUI {
     /**
      * Gets the annotation button element.
      *
+     * @param {string} customAnnotatorSelector Class selector for a custom annotation button.
      * @return {HTMLElement} Annotate button element
      */
-    getAnnotateButton() {
-        return this.container.querySelector(SELECTOR_BOX_PREVIEW_BTN_ANNOTATE);
+    getAnnotateButton(customAnnotatorSelector) {
+        const selector = customAnnotatorSelector || SELECTOR_BOX_PREVIEW_BTN_ANNOTATE_POINT;
```

Comment:
Did this to avoid breaking tests that called getAnnotateButton(), would it be better to just use customAnnotatorSelector?